packages/sanity/src/structure/panes/documentList/sheetList/useDocumentSheetColumns.tsx

```diff
@@ -126,18 +127,17 @@ export function useDocumentSheetColumns(documentSchemaType?: ObjectSchemaType) {
         id: 'selected',
         enableHiding: false,
         header: (info) => (
-          <Checkbox
-            indeterminate={info.table.getIsSomeRowsSelected()}
-            onChange={info.table.getToggleAllPageRowsSelectedHandler()}
-          />
-        ),
-        cell: (info) => (
-          <Checkbox
-            checked={info.row.getIsSelected()}
-            disabled={!info.row.getCanSelect()}
-            onChange={() => info.row.toggleSelected()}
-          />
+          <Box>
+            <Checkbox
+              style={{paddingLeft: 4}}
+              indeterminate={info.table.getIsSomeRowsSelected()}
+              onChange={info.table.getToggleAllPageRowsSelectedHandler()}
+            />
+            {/* eslint-disable-next-line i18next/no-literal-string */}
```

Comment:
I wonder if `selected` should be added to the attribute exceptions list

Reply:
This was just some text I'm printing to the DOM so we can make sure that the correct rows are being selected. It's quite likely this would be deleted when we start improving styling